packages/run-protocol/src/vpool-xyk-amm/params.js

```diff
@@ -15,35 +10,11 @@ const DEFAULT_POOL_FEE_BP = 24n;
 const DEFAULT_PROTOCOL_FEE_BP = 6n;
 
 /**
- * @param {ERef<ZoeService>} zoe
- * @param {bigint} poolFeeBP
+ * @param {Invitation} electorateInvitationAmount - invitation for the question poser
```

Comment:
ouch. That hurts my brain. Is it an invitation or an amount? It's an invitation, right?

Reply:
this is just matching the existing code but I'll strike "Amount"